packages/zowe-explorer/src/tools/ZoweTerminal.ts

```diff
@@ -122,7 +132,9 @@ export class ZoweTerminal implements vscode.Pseudoterminal {
     private closeEmitter = new vscode.EventEmitter<void>();
     public onDidClose?: vscode.Event<void> = this.closeEmitter.event;
 
-    public open(_initialDimensions?: vscode.TerminalDimensions | undefined): void {
+    public open(initialDimensions?: vscode.TerminalDimensions | undefined): void {
+        this.mCols = initialDimensions?.columns ?? 80;
+
         this.writeLine(this.chalk.dim.italic(this.mMessage));
         if (this.command.length > 0) {
```

Comment:
I know there weren't changes to this in this PR, but I'm wondering if we should quickly address the lint error here by changing line 140 to `void this.handleInput(ZoweTerminal.Keys.ENTER);`

Reply:
Resolved in 0526544f1d6597ed18b3ed0f016f9574be47d072